src/main/java/nwoolcan/model/utils/Quantity.java

```diff
@@ -64,7 +64,7 @@ public int compareTo(final Quantity other) {
         if (this.equals(other)) {
             return 0;
         }
-        return this.getValue() < other.getValue() ? -1 : 1;
+        return Integer.compare(getValue(), other.getValue());
```

Comment:
Should not be in this branch

Reply:
I know but i was too lazy to create another for one line of code, sorry